server/db/seeds/player.js

```diff
@@ -1,3 +1,6 @@
+// "pass" in plaintext
+const password = '$2a$10$vcJcMadI89UQoQjcSmgnGejKRryJb2/jAgHrqpm1pTfsBLumSdtSG'
```

Comment:
Is this information sensitive? If so, then this should be read from an untracked configuration file. Be careful with checking in credentials.

Reply:
The seeds are just for development purposes; I wouldn't expect this data to ever make its way into a production database.